src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -8,6 +8,7 @@
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/kubernetes-client/csharp</PackageProjectUrl>
     <PackageIconUrl>https://raw.githubusercontent.com/kubernetes/kubernetes/master/logo/logo.png</PackageIconUrl>
```

Comment:
remove?

Reply:
From the document, we could keep this for better compatibility. However, it may not make much sense. I would remove it if you want.

https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packageicon

> To maintain backward compatibility with clients and sources that don't yet support PackageIcon, specify both PackageIcon and PackageIconUrl